src/misc/ssEconomy.lua

```diff
@@ -218,9 +224,30 @@ function ssEconomy:calculateLoanInterestRate()
     g_currentMission.missionStats.loanAnnualInterestRate = seasonsYearInterest
 end
 
+function ssEconomy:aiLoad(savegame)
+
+	-- After loading the aiVehicle, store original pricePerMS in a variable, so pricePerMS can be altered without losing the original value.
+	self.ssOriginalPricePerMS = self.pricePerMS 
+end
+
 function ssEconomy.aiUpdateTick(self, superFunc, dt)
-    if self:getIsActive() and self.pricePerMS ~= 0 then
-        self.pricePerMS = ssUtil.isWorkHours() and g_seasons.economy.aiPricePerMSWork or g_seasons.economy.aiPricePerMSOverwork
+    if self:getIsActive() then
+    	if self.ssOriginalPricePerMS ~= nil then
+
+    		-- Only apply the multiplier when price is positive, to avoid increase in 'worker income' in overtime
+    		if self.ssOriginalPricePerMS >= 0 then
+    			local factor;
+    			if ssUtil.isWorkHours() then
```

Comment:
Your choice:

this part can be:
`local factor = ssUtil.isWorkHours() and g_seasons.economy.aiPriceFactor or g_seasons.economy.aiPriceOverworkFactor`

Like below.

I don't know what is clearer, really...